src/facade/dragonfly_connection.cc

```diff
@@ -279,8 +279,10 @@ Connection::Connection(Protocol protocol, util::HttpListenerBase* http_listener,
   last_interaction_ = creation_time_;
   id_ = next_id.fetch_add(1, memory_order_relaxed);
 
-  if (queue_backpressure_.limit == 0)
-    queue_backpressure_.limit = absl::GetFlag(FLAGS_pipeline_queue_limit);
+  queue_backpressure_ = &tl_queue_backpressure_;
+  if (queue_backpressure_->limit == 0) {
```

Comment:
dead if :)

Reply:
oh you fixed it :)

Reply:
> dead if :)

Yeah that was a demonstration of my quick auto-replace capabilities 🤣